### ISSUE TITLE
[BUG] - certain plugins fail to install

### ISSUE THREAD

**Issue by jdewsnip** [k3ai/k3ai#22]
**Describe the bug**

Think the install dir could be hard coded?

**To Reproduce**

1. k3ai plugin deploy -n kf-pa -t mycluster 

**Expected behavior**

installs ok like the mlflow one

**Screenshots**

```
fermi ॐ  ~:
2112 ☿ k3ai plugin deploy -n kf-pa -t mycluster                                                                                                                         ⏎

**mkdir /home/alefesta**: permission denied 🚀 Starting installation...
 
 ⏳	Working... 
 🚀 error: evalsymlink failure on '/home/john/.k3ai/git/apps/pipeline/upstream/cluster-scoped-resources' : lstat /home/john/.k3ai/git: no such file or directory
 🚀 Working on the installation...
```

**Comment by alefesta** [comment]
Ah damn... that's definitely my fault! Thank you @jdewsnip for spotting the problem.. yep it was in the git command..let's call it a leftover from the final party release 😅 

Going to fix it and update the release.

**Comment by alefesta** [comment]
Fixed through #23 PR. You should re-download the binary, to do so you may use:
```bash
curl -sfL https://get.k3ai.in | sh -
```
No need to re-run the `k3ai up`
Let me know if everything works...I'll close the issue but feel free to comment or re-open it if doesn't work.
Again many thanks for your issue!!!!
